specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-04-01/registries.json

```diff
@@ -0,0 +1,911 @@
+    "AcrDetails": {
+      "description": "Details of ACR account to be used for the Registry",
+      "type": "object",
+      "properties": {
+        "systemCreatedAcrAccount": {
+          "description": "Details of system created ACR account to be used for the Registry",
+          "$ref": "#/definitions/SystemCreatedAcrAccount",
+          "x-nullable": true
+        },
+        "userCreatedAcrAccount": {
+          "description": "Details of user created ACR account to be used for the Registry",
+          "$ref": "#/definitions/UserCreatedAcrAccount",
+          "x-nullable": true
+        }
+      },
+      "additionalProperties": false
+    },
+    "ArmResourceId": {
+      "description": "ARM ResourceId of a resource",
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "description": "Arm ResourceId is in the format \"/subscriptions/{SubscriptionId}/resourceGroups/{ResourceGroupName}/providers/Microsoft.Storage/storageAccounts/{StorageAccountName}\"\r\nor \"/subscriptions/{SubscriptionId}/resourceGroups/{ResourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{AcrName}\"",
+          "type": "string",
+          "x-nullable": true
+        }
+      },
+      "additionalProperties": false
+    },
+    "EndpointServiceConnectionStatus": {
+      "description": "Connection status of the service consumer with the service provider",
+      "enum": [
+        "Approved",
+        "Pending",
+        "Rejected",
+        "Disconnected"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "EndpointServiceConnectionStatus",
+        "modelAsString": true
+      },
+      "additionalProperties": false
+    },
+    "PartialManagedServiceIdentity": {
+      "description": "Managed service identity (system assigned and/or user assigned identities)",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Managed service identity (system assigned and/or user assigned identities)",
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentityType"
+        },
+        "userAssignedIdentities": {
+          "description": "The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/PartialUserAssignedIdentity"
+          }
+        }
+      },
+      "additionalProperties": false
+    },
+    "PartialRegistry": {
+      "description": "Partial Registry class for PATCH",
+      "type": "object",
+      "additionalProperties": false
+    },
+    "PartialRegistryPartialTrackedResource": {
```

Comment:
>PartialRegistryPartialTrackedResource [](http://example.com/codeflow?start=5&length=37)

recommend using types from common types (non blocking comment since this is already checked in) . This will avoid making mistakes like this.